src/sha1_hash.cpp

```diff
@@ -63,45 +62,6 @@ namespace libtorrent
 
 #endif // TORRENT_USE_IOSTREAM
 
-	int sha1_hash::count_leading_zeroes() const
-	{
-		int ret = 0;
-		for (int i = 0; i < number_size; ++i)
-		{
-			std::uint32_t v = aux::network_to_host(m_number[i]);
```

Comment:
hmmm...it is. I see you use it too in `bitfield::find_first_set` in your other PR. Any suggestion?